suite-common/wallet-networks-config/package.json

```diff
@@ -0,0 +1,12 @@
+{
+    "name": "@suite-common/wallet-networks-config",
```

Comment:
Why have the `wallet-network-config` and `wallet-constants` in separate packages? It's a package per file, which we didn't want, isn't it?

Reply:
With packages, I am mostly following folder structure that was used in suite:

`/suite/constants/wallet` => @suite-common/wallet-constants
`/suite/config/wallet` => @suite-common/wallet-networks-config (this is actually mistake, it makes more sense to name it just `wallet-config`, I will rename it).
`/suite/types/wallet` => @suite-common/wallet-types

About the one file per package, it can happen sometimes and there is nothing bad about it. Sometimes is just makes sense to have package with just one file. It absolutely same like to have folders with just one file, and we have lot of them now. (for example `/suite/config/wallet`).

I don't want to change how files are structures in this PR. I just want to make packages from them a keep changes to minimum. It will be better to do refactoring later and it's actually again easier with smaller packages, you can refactor inside only one small package, it's easier to merge two small packages later that split one big into smaller, it's easier to move packages around if they are small (less fixing of imports, less diff etc.)...

Reply:
Renamed to just `@suite-common/wallet-config`

Reply:
You think that having a package per domain would be a worse move? `@suite-common/wallet` for all the types, constants, configs, etc? If you are a package that needs to use something related to wallets you will likely use not _just_ types or _just_ constants. So to me it's the same as having `@suite-native/module-home-root`, `@suite-native/module-home-state` and `@suite-native/module-home-constants`.

Reply:
You are actually right and per domain packages is what we want to do. Problem here is that current "wallet" domain is really big and it's something that we don't want all in one package. This something that we don't want, because packages are like files, if you have file few hundreds of thousands lines in one file you know it is really bad and it's same for packages. I know config + constants + types are quite small, but in next PR we will add also wallet utils which would cause this one wallet package to be like really big and have probably hundreds of exported functions, types and constants. That would be too much.

Spliting giant wallet package also per type (config, constants, types, utils) is actually little bit of trick how to reduce package size.

In ideal world we will have small domain packages for everything in wallet. For example we would have `wallet-transactions` package that will contains everything for transactions (types, utils, reducers...), but that's something that is impossible for us now. It's super hard to migrate to this kind structure because everything depends on everything. For example transactions depends on account stuff, that accounts depends on other stuff etc. etc. and in the end we will need to go very deep and create like 10 packages with new structure than current `/suite` just because of transactions. That would be really overhelming and it will took maybe weeks to do it properly.